src/core_atmosphere/physics/mpas_atmphys_driver_sfclayer.F

```diff
@@ -375,12 +453,20 @@ subroutine sfclayer_from_MPAS(configs,mesh,diag_physics,sfc_input,its,ite)
                                          qgh,qsfc,lh,mol,psim,psih,regime,rmol,ust,ustm, &
                                          wspd,znt,zol
 
+
 !local pointers specific to monin_obukhov:
  real(kind=RKIND),dimension(:),pointer:: fh,fm
 
 !local pointers specific to mynn:
  real(kind=RKIND),dimension(:),pointer:: ch,qcg,snowh
-
+ real(kind=RKIND),dimension(:,:),pointer:: cov,el_pbl,qsq,sh3d,tsq
```

Comment:
this line should be removed. we no longer need to have cov,el_pbl,qsq,sh3d,tsq in this driver